Lib/test/test_asyncio/test_base_events.py

```diff
@@ -1286,9 +1285,7 @@ def _test_create_connection_ip_addr(self, m_socket, allow_inet_pton):
         sock = m_socket.socket.return_value
 
         self.loop._add_reader = mock.Mock()
-        self.loop._add_reader._is_coroutine = False
```

Comment:
when asyncio.iscoroutinefunction was implemented as:

https://github.com/python/cpython/blob/2f841442354f41f2490cef76e9ba24bfef9dfbea/Lib/asyncio/coroutines.py#L245-L248

this meant
```python
self.loop._add_reader = mock.Mock()
assert asyncio.iscoroutinefunction(self.loop._add_reader) is True  # mock objects have a truthy _is_coroutine attribute!
self.loop._add_reader._is_coroutine = False
assert asyncio.iscoroutinefunction(self.loop._add_reader) is False  # patching it works around the issue
```

however the implementation was changed to use a marker object:

https://github.com/python/cpython/blob/ec4745b5f2c2a460f12c97243c5b7d875f24b0c8/Lib/asyncio/coroutines.py#L16-L23

and so now the `_is_corutine` workaround is redundant:

```python
self.loop._add_reader = mock.Mock()
assert asyncio.iscoroutinefunction(self.loop._add_reader) is False  # mock objects have an _is_coroutine but it's not the asyncio.coroutines._is_coroutine sentinel
self.loop._add_reader._is_coroutine = False
assert asyncio.iscoroutinefunction(self.loop._add_reader) is False  # it's still False so the workaround is redundant
```